src/manifestmerger/build.gradle

```diff
@@ -19,7 +19,7 @@ repositories {
 
 dependencies {
     // https://mvnrepository.com/artifact/com.android.tools.build/manifest-merger
-    compile group: 'com.android.tools.build', name: 'manifest-merger', version: '30.3.1'
+    implementation 'com.android.tools.build:manifest-merger:30.3.1'
```

Comment:
This looks right, like what `Gradle (short)` shows here: 

https://mvnrepository.com/artifact/com.android.tools.build/manifest-merger/30.3.1